dev_cycle/terraform/main.tf

```diff
@@ -69,6 +69,21 @@ data "aws_iam_policy_document" "build_machine" {
     ]
   }
 
+  statement {
+    effect = "Allow"
+    actions = [
+      "kms:CreateGrant",
+      "kms:Encrypt",
+      "kms:Decrypt",
+      "kms:ReEncrypt*",
+      "kms:GenerateDataKey*",
+      "kms:DescribeKey",
+    ]
+    resources = [
+      "arn:aws:kms:*:*:key/${var.ebs_kms_key}"
```

Comment:
I think we should set the entire ARN here instead of only the last part. By default working with ARNs gives you greater control over the permissions.

Reply:
changed to arn